Sources/SmithyFormURL/NodeInfo.swift

```diff
@@ -0,0 +1,24 @@
+//
+// Copyright Amazon.com Inc. or its affiliates.
+// All Rights Reserved.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+public struct NodeInfo: Equatable {
```

Comment:
This is `NodeInfo` for the FormURL writer.  Since form URL has no namespaces, etc. like XML, this type only holds a name.